liberty-maven-plugin/src/it/dev-it/resources/multi-module-projects/multipleLibertyModules/ear2/src/main/liberty/config/server.xml

```diff
@@ -0,0 +1,16 @@
+<server description="Sample Liberty server">
+
+    <featureManager>
+        <feature>jsp-2.3</feature>
+    </featureManager>
+
+    <variable name="default.http.port" defaultValue="9080" />
```

Comment:
Would we ever need to run the ear1 and ear2 servers at the same time?

Reply:
We can consider adding that scenario separately and adjust the value if needed.

Reply:
Opened a separate issue to test this. https://github.com/OpenLiberty/ci.maven/issues/1188